extensions/attention-stream/src/background.ts

```diff
@@ -661,5 +665,40 @@ if (enableDevMode) {
   } else {
     browser.action.onClicked.addListener(actionListener);
   }
+
+  // Show existing users a new tab page, on update only.
+  browser.runtime.onInstalled.addListener(async (details) => {
+    if (details.reason === "update") {
+      const idleListener = async (newState: browser.Idle.IdleState) => {
+        if (newState !== "idle") {
+          return false;
+        }
+
+        // Try to avoid interrupting the user if they are playing media on the currently-focused tab.
```

Comment:
@aaga See the comments in the PR for details, but note that I took your advice on removing the initial check for idle state (to simplify things and remove the possibility for bugs like the one you found), but I think we probably want to keep the idle timer since it'll avoid user input.

When testing this though, I noticed that it also interrupts videos, so I added this simple check to see if media is playing and to put off opening the new tab in that case as well.